libraries/botbuilder-azure-blobs/src/blobsStorage.ts

```diff
@@ -36,8 +36,7 @@ export class BlobsStorage implements Storage {
      * @param {BlobsStorageOptions} options Other options for BlobsStorage
      */
     constructor(connectionString: string, containerName: string, options?: BlobsStorageOptions) {
-        assert.string(connectionString, ['connectionString']);
-        assert.string(containerName, ['containerName']);
+        t.Record({ connectionString: t.String, containerName: t.String }).check({ connectionString, containerName });
```

Comment:
Publicly asking: is the error message thrown from this straight forward?

Reply:
Yes - the Record wrapping is to include the key in the error message:
```
> t.String.check(10)
Uncaught ValidationError: Expected string, but was number
    at new ValidationError (/Users/jpg/src/github/microsoft/botbuilder/js/node_modules/runtypes/lib/errors.js:22:28)
    at Object.check (/Users/jpg/src/github/microsoft/botbuilder/js/node_modules/runtypes/lib/runtype.js:30:15) {
  key: undefined
}
> t.Record({foo: t.String}).check({foo: 10})
Uncaught ValidationError: Expected string, but was number in foo
    at new ValidationError (/Users/jpg/src/github/microsoft/botbuilder/js/node_modules/runtypes/lib/errors.js:22:28)
    at Object.check (/Users/jpg/src/github/microsoft/botbuilder/js/node_modules/runtypes/lib/runtype.js:30:15) {
  key: 'foo'
}
```

Reply:
Unfortunately, using `.withBrand('foo')` doesn't include it anywhere in the error message.